File: Data Structures & Algorithms/divide-array-into-equal-pairs/submission-0.go
func divideArray(nums []int) bool {
    seen := make(map[int]int)
	for _, val := range nums {
		seen[val]++
	}

	for _,val := range seen {
		if val % 2 != 0 {
			return false
		}
	}
	
	return true
}

File: Data Structures & Algorithms/count-the-number-of-consistent-strings/submission-0.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        seen = {}
        count = len(words)
        for char in allowed:
            seen[char] = True

        for word in words:
            for i in range(len(word)):
                if word[i] not in seen:
                    count -= 1
                    break

        return count